examples/customize-extractor-error/Cargo.toml

```
@@ -2,12 +2,16 @@
 name = "example-customize-extractor-error"
 version = "0.1.0"
 edition = "2021"
-publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

We'll wanna keep this so we don't accidentally publish this crate.

```suggestion
publish = false
```